### PR TITLE
[json-rpc] fix event endpoint

### PR DESCRIPTION
**Pull request by phoenix-antigravity**
* return events in ascending order in json-rpc
* fix `q ev` command in CLI

### CODE REVIEW

**Comment by thefallentree** [comment]
This should have been caught by the test right?

File: storage/libradb/src/lib.rs [hunk 1]
```diff
@@ -898,7 +898,7 @@ impl LibraDBTrait for LibraDB {
             .ledger_info()
             .version();
         let events = self
-            .get_events_by_event_key(event_key, start, false, limit, version)?
+            .get_events_by_event_key(event_key, start, true, limit, version)?
```
Comment (zekun000) [inline storage/libradb/src/lib.rs:901]: this breaks how we get the latest events from this endpoint (start == u64::max_value) (which breaks the leader reputation algorithm)

Comment (phoenix-antigravity) [reply storage/libradb/src/lib.rs:901]: @zekun000: can you refer to call site? Why it uses LibraDBTrait at all?

**Pull request closed without merging**